### DISCUSSION TITLE
graph复制一个React组件节点时，粘贴时React组件节点的component为null

### DISCUSSION THREAD

**Discussion by cuiyiworld** [antvis/X6#2133] [Q&A]
graph复制一个React组件节点时，粘贴时React组件节点的component为null。

**Comment by NewByVector** [comment]
是不是 react 节点注册方式不对，要能复制粘贴的话，需要节点数据能够序列化。

**Reply by cuiyiworld** [reply]
我是以这种方式注册的，
<img width="781" alt="image" src="https://user-images.githubusercontent.com/21233144/170397375-23bc6651-2bff-4979-a5bb-a6386c520ec3.png">
这种方式如果不行的话，是需要用 Graph.registerNode 或 Graph.registerReactComponent 方式注册吗？

**Reply by NewByVector** [reply]
这种不行的，使用 `Graph.registerNode` 吧。

**Reply by cuiyiworld** [reply]
好的，谢谢。
